scripts/release_notes.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/env bash
+
+if [ ! -n "$GH_USER" ] || [ ! -n "$GH_TOKEN" ]; then
+    echo "You must export GH_USER - your github user id"
+    echo "You must export GH_TOKEN - your github access token"
+    exit 1
+fi
+
+if [ ! -n "$1" ]; then
+    echo "Must provide arg"
```

Comment:
I'd say what the arg is. I.e. "Must provide start commit sha" or something more explicit. Otherwise I'll just enter "fu"